client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -51,9 +56,43 @@ class GoogleMyBusinessSelectBusinessType extends Component {
 		);
 	};
 
+	handleConnect = () => {
+		/* eslint-disable no-console */
+		console.log( 'connected' );
+		// TODO: handle redirect to the "Create My Listings" page here
```

Comment:
It would be good to rebase this pull request against `master` since we already have a link in place that goes to the `New Account` page (and it would be better to keep it so it's easy to navigate the full flow).